app/views/katello/subscription_mailer/subscription_expiry.html.erb

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" style="font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; color: #3f3f3f; background-color: #f1f1f1;">
+<body>
+<style type="text/css">
+  a, a:visited {
+    color: #6bb5df !important;
+  }
+
+  a:hover {
+    color: #CCCCCC !important;
+  }
+</style>
+<div style="font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; color: #3f3f3f; background-color: #f1f1f1;">
+  <h2 style="font-weight: normal;"><%= _("SUBSCRIPTIONS EXPIRING SOON").html_safe %></h2>
+    <table border="0" width="100%" id="emailBody" style="background-color: #fff; margin-top: 10px; padding: 0px 0px 15px;" bgcolor="#fff">
+      <tr>
+        <th style="text-align: center; border-collapse: collapse; padding: 4px; border: 1px solid #cccccc;" align="center"><%= _("Subscription Name") %></th>
+        <th style="text-align: center; border-collapse: collapse; padding: 4px; border: 1px solid #cccccc;" align="center"><%= _("Account Number") %></th>
+        <th style="text-align: center; border-collapse: collapse; padding: 4px; border: 1px solid #cccccc;" align="center"><%= _("Organization") %></th>
+        <th style="text-align: center; border-collapse: collapse; padding: 4px; border: 1px solid #cccccc;" align="center"><%= _("Type") %></th>
+        <th style="text-align: center; border-collapse: collapse; padding: 4px; border: 1px solid #cccccc;" align="center"><%= _("Quantity") %></th>
+        <th style="text-align: center; border-collapse: collapse; padding: 4px; border: 1px solid #cccccc;" align="center"><%= _("SKU") %></th>
+        <th style="text-align: center; border-collapse: collapse; padding: 4px; border: 1px solid #cccccc;" align="center"><%= _("Contract") %></th>
+        <th style="text-align: center; border-collapse: collapse; padding: 4px; border: 1px solid #cccccc;" align="center"><%= _("Subscription Start") %></th>
```

Comment:
These headings are not consistent with our conventions in the UI: 'Subscription Start', 'Subscription End', 'Subscription Ends In'

Since we're looking at a table of subscriptions we don't need to use the word 'Subscription' in the heading, so Start Date and End Date will do - just like in the UI.

For 'Subscription Ends In'  I think a better heading would be 'Days remaining' to give the unit of measure

Reply:
@jturel Thank you for your suggestions. I have made the required changes.

Reply:
@jturel The headings in the [report template PR](https://github.com/theforeman/community-templates/pull/705) are 'Subscription Start', 'Subscription End', and 'Subscription Expires In' (I had changed 'Ends In' to 'Expires In' to address an @ares comment.)  With that exception, the recent work didn't change any of the headings.  Should these headings be changed to be consistent across both locations?

Reply:
Consistency is good! Although, that table is showing a mix of host and subscription data, so the 'subscription' qualifier in the heading names is helpful there. 

As far as the report using the word 'days' as part of the value in a particular row: I think that  should be changed. It's unusual to include the unit of measure in the value rather than the field/column name. If someone generates the report in CSV or JSON format to parse with some custom tool they basically have to do post-processing on each row to get the numeric value apart from 'days'.

Reply:
@jturel ok, changed the community-templates column heading to 'Days Remaining'.

Reply:
Lol, nice. Thought that got merged in already!